app/views/layouts/application.html.erb

```diff
@@ -18,7 +18,11 @@
       <% end %>
       <meta name="environment" content="<%= Rails.env %>">
       <%= render "layouts/styles", qualifier: "main" %>
-      <%= javascript_include_tag "base", "application", "baseInitializers", "Search", "runtimeBanner", "baseTracking", defer: true %>
+      <% if Settings::UserExperience.show_mobile_app_banner %>
+        <%= javascript_include_tag "base", "application", "baseInitializers", "Search", "runtimeBanner", "baseTracking", defer: true %>
+      <% else %>
+        <%= javascript_include_tag "base", "application", "baseInitializers", "Search", "baseTracking", defer: true %>
+      <% end %>
```

Comment:
nit: Is there benefit to keeping it all in a single `javascript_include_tag`? I figure it would be easier to understand this way.

```
      <% if Settings::UserExperience.show_mobile_app_banner %>
        <%= javascript_include_tag "runtimeBanner", defer: true %>
      <% else %>
```

Reply:
Yeah good call, much better, making the chaneg.